apps/api/v2/src/modules/verified-resources/verified-resources.repository.ts

```diff
@@ -0,0 +1,104 @@
+import { PrismaReadService } from "@/modules/prisma/prisma-read.service";
+import { PrismaWriteService } from "@/modules/prisma/prisma-write.service";
+import { Injectable } from "@nestjs/common";
+
+@Injectable()
+export class VerifiedResourcesRepository {
+  constructor(private readonly dbRead: PrismaReadService, private readonly dbWrite: PrismaWriteService) {}
+
+  async getUserVerifiedEmails(userId: number, skip = 0, take = 250) {
```

Comment:
I would split this into `UserVerifiedResourcesRepository` and `TeamVerifiedResourcesRepository`